Lib/ftplib.py

```diff
@@ -529,12 +532,12 @@ def storlines(self, cmd, fp, callback=None):
         return self.voidresp()
 
     def acct(self, password):
-        '''Send new account name.'''
+        """Send new account name."""
         cmd = 'ACCT ' + password
         return self.voidcmd(cmd)
 
     def nlst(self, *args):
-        '''Return a list of files in a given directory (default the current).'''
+        """Return a list of files in a given directory (default the current)."""
```

Comment:
https://www.python.org/dev/peps/pep-0008/#maximum-line-length

It said `For flowing long blocks of text with fewer structural restrictions (docstrings or comments), the line length should be limited to 72 characters.`

It looks like a good time to update docstring to follow this limit.

Reply:
@alanyee Please follow @aeros167 's feedback.
I think that @aeros167  's feedback is the right way for this moment.